1.8/administration/overlay-networks/ip-per-container.md

```diff
@@ -4,30 +4,28 @@ post_title: Configuring IP-per-Container in Overlay Networks
-        prefix: 24
-      - name: dcos-2
-        subnet: 192.168.128.0/17
-        prefix: 24
+      - name: dcos
+        subnet: 9.0.0.0/8
+        prefix: 26
 ```
 
 **Note:** Use a recent Linux kernel (3.9 or later) and Docker version 1.11 or later on the agent nodes.
 
-Each overlay network is identified by a canonical `name`. Containers launched on an overlay network will get an IP address from the subnet allocated to the overlay network. To remove the dependency on a global IPAM, the overlay subnet is further split into smaller container subnets. Each of the container subnets is allocated to an agent. The agents can then use a host-local IPAM to allocate IP addresses from their respective container subnets to containers launched on the agent and attached to the given overlay. The `prefix` determines the size of the container subnet (carved from the overlay subnet) allocated to each agent and thus defines the number of agents on which the overlay can run.
+Each overlay network is identified by a canonical `name` (see [limitations](#limitations) for constraints on naming overlay networks). Containers launched on an overlay network will get an IP address from the subnet allocated to the overlay network. To remove the dependency on a global IPAM, the overlay subnet is further split into smaller subnets. Each of the smaller subnets is allocated to an agent. The agents can then use a host-local IPAM to allocate IP addresses from their respective subnets to containers launched on the agent and attached to the given
+overlay. The `prefix` determines the size of the subnet (carved from the overlay subnet) allocated to each agent and thus defines the number of agents on which the overlay can run.
 
-In the configuration above each overlay network is allocated a /17 subnet (in the “subnet” field), which is then divided into /24 container subnets to be used on each host that will be part of the network (in the “prefix” field) as shown:
+In the configuration above each overlay network is allocated a /8 subnet (in the “subnet” field), which is then divided into /26 container subnets to be used on each host that will be part of the network (in the “prefix” field) as shown:
 
 ![Overlay network address space](/1.8/administration/overlay-networks/img/overlay-network-address-space.png)
 
-These values can be configured to adapt to each installation’s needs. If you need a larger network, decrease the network mask prefix of the `subnet`. To create an overlay network for only one agent node, specify the same value for the `subnet` network mask and the `prefix`. The size of the container space is `2^(32-<prefix>)`.
+These values can be configured to adapt to each installation’s needs. If you need to change the network size, modify the network mask prefix of the `subnet`. To create an overlay network for only one agent node, specify the same value for the `subnet` network mask and the `prefix`. The size of the container space is `2^(32-<prefix>)`.
```

Comment:
I think you should write `2^5 = 32`.
People don't search documentation for the functions that result in the errors they hit. They search for the actual values.